eodatasets3/properties.py

```diff
@@ -180,6 +180,7 @@ def parsed_sentinel_tile_id(tile_id) -> Tuple[str, Dict]:
     "landsat:image_quality_oli": None,
     "landsat:image_quality_tirs": None,
     "landsat:processing_software_version": None,
+    "landsat:scan_gap_interpolation": float,
```

Comment:
Can you confirm if *float* is correct for the scan gap interpolation property?  I thought it'd be a boolean or something.

Reply:
Is it number of pixels or something?

Reply:
Yeah, in the lpgs MTLs it's a float. In our case, they're all either '2.0' (usgs) or '0.0' (ours). A boolean is probably more in line with how we use the field ourselves today, but it didn't seem worth deviating from the norm.